lamindb/models/schema.py

```diff
@@ -626,11 +627,27 @@ def registry(self) -> str:
     def registry(self, value) -> None:
         self.itype = value
 
+    @property
+    def slots(self) -> dict[str, Schema]:
+        """Slots."""
+        if hasattr(self, "_slots"):
+            return self._slots
+        if self.itype == "Composite":
+            self._slots = {
```

Comment:
This isn't tested.

Reply:
It should also replace the `_get_component(slot: str)` method. If you refactor this it's also going to be tested.